tests/v3/test_group.py

```diff
@@ -390,6 +390,64 @@ def test_group_create_array(
     assert np.array_equal(array[:], data)
 
 
+def test_group_array_creation(
+    store: Store,
+    zarr_format: ZarrFormat,
+):
+    group = Group.create(store, zarr_format=zarr_format)
+    shape = (10, 10)
+
+    empty_array = group.empty(shape=shape)
+    assert isinstance(empty_array, Array)
+    assert empty_array.fill_value == 0
+    assert empty_array.shape == shape
```

Comment:
Fill value expected to be 0: https://github.com/zarr-developers/zarr-python/blob/v3/src/zarr/core/array.py#L253-L257